src/payment/klarna-setup.md

```diff
@@ -31,96 +31,91 @@ _Klarna Merchant Portal_
+   _Generate API Credentials_
 
 ## Step 2: Configure Magento
 
-1. Open another browser tab, and log in to the Admin of your Magento store.
+1. In a new browser tab, log in to the Admin of your Magento store.
 
-1. On the Admin sidebar, choose **Stores**. Then under **Settings**, choose **Configuration**.
+1. On the Admin sidebar, go to **Stores** > `Settings`> **Configuration**.
 
 1. In the upper-left corner, choose the **Store View** where the configuration applies. If your installation has only one view, accept the `Default Config` setting.
 
-1. In the panel on the left under **Sales**, choose **Payment Methods**. In the **Klarna** section, click <span class="btn">Configure</span>.
+1. In the left panel, go to **Sales** > **Payment Methods**. Then in the **Klarna** section, click <span class="btn">Configure</span>.
 
    ![Klarna]({% link images/images/config-sales-payment-methods-recommended-solutions-klarna.png %}){: .zoom}
    [_Klarna_]({% link configuration/sales/klarna.md %})
 
-1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **API Configuration** section. Then, do the following:
+1. Expand ![]({% link images/images/btn-expand.png %}){: .Inline} the **API Configuration** section.
+
+   ![API Configuration]({% link images/images/config-sales-payment-methods-klarna-api.png %}){: .zoom}
+   [_API Configuration_]({% link configuration/sales/klarna.md %})
+
+1. Set **Endpoint** to one of the following:
+
+   - **Klarna Payments (North America)**
+   - **Klarna Payments (Europe)**
+   - **Klarna Payments (Oceana)**
 
-   - Set **Endpoint** to one of the following:
+   Klarna supports the following countries:  AT, DE, DK, FI, NL, NO, SE, US, AU, NZ
 
-      - **Klarna Payments (North America)**
-      - **Klarna Payments (Europe)**
-      - **Klarna Payments (Oceana)**
+1. In the **Merchant ID** field, enter the `API Username (UID)` from your Klarna API credentials.
 
-      Supported countries:  AT, DE, DK, FI, NL, NO, SE, US, AU, NZ
+   {:.bs-callout-info}
+   Do not enter the `Merchant ID` from your Klarna account.
 
-   - Enter the **Merchant ID** from your Klarna account.
-   - Enter the **Password** from your Klarna account.
-   - To use the Klarna Playground for test transactions, set **Mode** to `Playground`.
-   - To generate a record of interactions while testing the integration, set **Logging** to `Enable`.
+1. Enter the **Password** from your Klarna API credentials.
 
-     {:.bs-callout-info}
-     Set **Logging** to `Disabled` before going live with Klarna in your production store.
+1. To use the Klarna Playground for test transactions, set **Mode** to `Playground`.
 
-     ![API Configuration]({% link images/images/config-sales-payment-methods-klarna-api.png %}){: .zoom}
-     [_API Configuration_]({% link configuration/sales/klarna.md %})
+1. To generate a record of interactions while testing the integration, set **Logging** to `Enable`.
```

Comment:
is it enabled and disabled or enable and disable?  the next line includes the "d"

Reply:
Enable is correct here.